README.md

```diff
@@ -40,6 +40,7 @@ This library currently supports the following cryptocurrencies and address forma
  - XRP (base58check-ripple)
  - BCH (base58check and cashAddr; decodes to cashAddr)
  - BNB (bech32)
+ - EGLD (bech32)
```

Comment:
Can you provide reference link so that we can verify?

Reply:
@makoto https://github.com/trustwallet/wallet-core/blob/8d3100f61e36d1e928ed1dea60ff7554bba0db16/tests/Elrond/AddressTests.cpp

Reply:
ah sorry, I missed out your initial comment